substrate/client/cli/src/config.rs

```diff
@@ -185,7 +185,12 @@ pub trait CliConfiguration<DCV: DefaultConfigurationValues = ()>: Sized {
 			)
 		} else {
 			NetworkConfiguration::new(node_name, client_id, node_key, Some(net_config_dir))
-		})
+		};
+
+		// TODO: Return error here in the next release.
```

Comment:
We should have an issue for this and link the place where to add IMO. Already too many dead TODOs in the code :grimacing: